test/remote/gateways/remote_rapyd_test.rb

```diff
@@ -196,6 +196,36 @@ def test_failed_verify
     assert_equal 'Do Not Honor', response.message
   end
 
+  def test_successful_store_and_unstore
+    store = @gateway.store(@credit_card, @options)
+    assert_success store
+    assert customer_id = store.params.dig('data', 'id')
+    assert store.params.dig('data', 'default_payment_method')
+
+    # payment_token = store.params.dig('data', 'payment_methods', 'data', 0, 'id')
+    # purchase = @gateway.purchase(@amount, payment_token, {})
+    # assert_success purchase
```

Comment:
LMK if I should add a clarifying comment on this or remove it completely.

Reply:
Yes, I think one or the other. I would say it was a good idea initially, but considering that it doesn't work as part of the automated test my vote would be to remove it completely.

If you disagree, then yes I think at the least a comment on why this is here and how to use it would be appropriate.

Reply:
Sounds good, I'd rather not have commented out code added to the main branch (that I'm pretty sure will just get forgotten 😞 )